endoreg_db/management/commands/import_video.py

```
@@ -351,11 +336,11 @@ def _ensure_default_patient_data(self, video_file):
             default_data = {
-                "patient_first_name": "Patient",
-                "patient_last_name": "Unknown", 
-                "patient_dob": date(1990, 1, 1),  # Default DOB
-                "examination_date": date.today(),
-                "center_name": video_file.center.name if video_file.center else "university_hospital_wuerzburg"
+                "patient_first_name": None,
+                "patient_last_name": None, 
+                "patient_dob": None,
+                "examination_date": None,
+                "center_name": video_file.center.name if video_file.center else 
             }
```

Pylint (4.0.2)

[error] Parsing failed: 'invalid syntax (endoreg_db.management.commands.import_video, line 344)'

(E0001)